.github/workflows/builNTest.yaml

```diff
@@ -0,0 +1,64 @@
+name: Build and Test
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+      - develop
+
+jobs:
+  build-and-test-on-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: 21
+          distribution: oracle
+      - name: Build and Test with Gradle
+        run: |
+          ./gradlew
+          ./gradlew clean
+          ./gradlew build
+          ./gradlew run
+          ./gradlew transform
+          ./gradlew reset
+
+  build-and-test-on-ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: 21
+          distribution: oracle
```

Comment:
In general, are we sure we want to be building & testing using Oracle's Java and on Ubuntu?

The current runners are running with OpenJDK and on SUSE.
```
$ java --version
openjdk 17.0.13 2024-10-15
OpenJDK Runtime Environment (build 17.0.13+11-suse-150400.3.48.2-x8664)
OpenJDK 64-Bit Server VM (build 17.0.13+11-suse-150400.3.48.2-x8664, mixed mode, sharing)
```

Reply:
Good to know. This is new information for me. Are these runners on the server that you are talking about?

Reply:
Sorry, the transformer will be running on that server.  Was confusing GH runners & the website's work queue system.

Reply:
That said, it is probably wise to be using OpenJDK if possible, since that's what will be used, and given Oracle's licensing nonsense.

Reply:
I think the specific OS won't matter (shouldn't anyway). But let's use OpenJDK because Nebraska has banned Oracle Java.